From fae9e8e688591e1680a7c8da2d20770e28b75e9b Mon Sep 17 00:00:00 2001
From: Phil Sutter <psutter@redhat.com>
Date: Wed, 5 Apr 2017 15:18:12 +0200
Subject: [PATCH] man: ip-link: Specify min/max values for bridge slave
 priority and cost

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1374360
Upstream Status: iproute2.git commit 058d28b44c6cc

commit 058d28b44c6cc66e5cdce31e80dc7284ccb22017
Author: Phil Sutter <phil@nwl.cc>
Date:   Tue Apr 4 17:08:43 2017 +0200

    man: ip-link: Specify min/max values for bridge slave priority and cost

    The values are parsed as u16/u32, but kernel limits allowed values.

    Signed-off-by: Phil Sutter <phil@nwl.cc>
---
 man/man8/ip-link.8.in | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/man/man8/ip-link.8.in b/man/man8/ip-link.8.in
index e4e95b7..5c80666 100644
--- a/man/man8/ip-link.8.in
+++ b/man/man8/ip-link.8.in
@@ -1390,10 +1390,10 @@ is a number representing the following states:
 .BR 4 " (blocking)."
 
 .BI priority " PRIO"
-- set port priority (a 16bit unsigned value).
+- set port priority (allowed values are between 0 and 63, inclusively).
 
 .BI cost " COST"
-- set port cost (a 32bit unsigned value).
+- set port cost (allowed values are between 1 and 65535, inclusively).
 
 .BR guard " { " on " | " off " }"
 - block incoming BPDU packets on this port.
-- 
1.8.3.1

